packages/cli/fragments/fragment-leptos/_Cargo.toml

```diff
@@ -5,11 +5,13 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
-leptos = { version = "0.1.0", features = ["stable"] }
+leptos = { version = "0.2", features = ["stable"] }
 serde = { version = "1.0", features = ["derive"] }
-serde-wasm-bindgen = "0.4"
+serde-wasm-bindgen = "0.5"
 wasm-bindgen = { version = "0.2", features = ["serde-serialize"] }
 wasm-bindgen-futures = "0.4"
+js-sys = "0.3"
+web-sys = "0.3"
```

Comment:
I don't think these are needed

Reply:
`wasm_bindgen` need those.

Reply:
keep `js-sys` then and remove `web-sys` if it is not needed